fuzzing/statetest_maker.go

```diff
@@ -146,9 +146,9 @@ func NewGstMaker() *GstMaker {
 			// The ENV portion
 			Number:     1,
 			GasLimit:   0x26e1f476fe1e22,
-			Difficulty: big.NewInt(0x20000),
+			Difficulty: big.NewInt(0x200000),
```

Comment:
Why? `0x20000` is the minimum allowed difficulty

Reply:
It's encoding issue. Maybe I can fix it on my side, but the official state tests add leading zeros in the encoding so the total number of chars is even.